src/test/java/org/checkerframework/specimin/Issue38Test.java

```diff
@@ -0,0 +1,15 @@
+package org.checkerframework.specimin;
```

Comment:
I think there should be one more test runner (for the `unsolvedexception` test).

Reply:
I did not add this since it will fail because of the issue I discussed in the meeting. I will fix the dependent issue and add the testrunner.

Reply:
I don't think this is done @tahiat